osquery/tables/system/darwin/processes.cpp

```diff
@@ -314,6 +315,29 @@ void genProcArch(QueryContext& context, int pid, ProcessesRow& r) {
     r.cpu_type_col = -1;
     r.cpu_subtype_col = -1;
   }
+
+  if (archinfo.p_cputype == CPU_TYPE_X86_64) {
+    r.kind_col = "intel";
+  }
+
+  if (archinfo.p_cputype == CPU_TYPE_ARM64) {
+    r.kind_col = "apple";
```

Comment:
Why not `arm64`?

Reply:
Just wanted to emulate the column name in Activity Monitor. Happy to change it arm64